app/frontend/src/cljs/teet/project/land_view.cljs

```diff
@@ -448,7 +457,17 @@
                                         [common/count-chip {:label c}])]
                         :loading-state "-"}]
           (tr [:land-modal-page :files])])
-
+       (let [land-owners-opinions-count (count (owners-opinions unit))]
+         (if (zero? land-owners-opinions-count)
+           [typography/GreyText
```

Comment:
typography elements should not wrap non text content, as it's against html rules for divs to be inside paragraphs